packages/core/src/Stores/traders-hub-store.js

```diff
@@ -642,23 +642,17 @@ export default class TradersHubStore extends BaseStore {
         });
         return all_svg_acc.length > 1;
     };
+
     getShortCodeAndRegion(account) {
         let short_code_and_region;
-        if (this.is_real && !this.is_eu_user) {
+        if (this.is_real && !this.is_eu_user && account.landing_company_short) {
             const short_code =
-                account.landing_company_short &&
-                account.landing_company_short !== 'svg' &&
-                account.landing_company_short !== 'bvi'
-                    ? account.landing_company_short?.charAt(0).toUpperCase() + account.landing_company_short?.slice(1)
-                    : account.landing_company_short?.toUpperCase();
-
-            let region = '';
-            if (this.hasMultipleSVGAccounts()) {
-                region =
-                    account.market_type !== 'financial' && account.landing_company_short !== 'bvi'
-                        ? ` - ${this.getServerName(account)}`
-                        : '';
-            }
+                account.landing_company_short === 'svg' || account.landing_company_short === 'bvi'
+                    ? account.landing_company_short.toUpperCase()
+                    : account.landing_company_short.charAt(0).toUpperCase() + account.landing_company_short.slice(1);
+
+            const region = account.landing_company_short === 'svg' ? ` - ${this.getServerName(account)}` : '';
```

Comment:
we do not need to show server name for financial accounts, please include that condition too